modules/ROOT/pages/_partials/data-sync/js-client-conflict.adoc

```diff
@@ -15,16 +15,22 @@ The conflict resolution implementation requires the following additions to your
 
 Developers can either use the default conflict resolution implementations, or implement their own conflict resolution implementations using the conflict resolution mechanism.
 
-By default, when no changes are made on the same fields, the implementation will attempt to resend the modified payload back to the server. When changes on the server and on the client cover the same fields, one of the specified conflict resolution strategies can be used. The default strategy will apply client changes on top of the server side data. Developers can modify strategies to suit their needs.
+By default, when no changes are made on the same fields, the implementation attempts to resend the modified payload back to the server. 
+When changes on the server and on the client affect the same fields, one of the specified conflict resolution strategies can be used. 
+The default strategy applies client changes on top of the server side data. 
+Developers can modify strategies to suit their needs.
 
 [#working-with-conflicts-client]
-== Working with Conflict Resolution on the Client
+== Implementing conflict resolution on the client
 
-To enable conflict resolution, the server side resolvers must be configured to perform conflict detection. Detection can rely on many different implementations and return the conflict error back to the client. For more information about how to do this, please see <<#conflict-resolution-{context}, Server Side Conflict Detection>>.
+To enable conflict resolution, the server side resolvers must be configured to perform conflict detection. 
+Detection can rely on different implementations and return the conflict error back to the client. 
+See <<#conflict-resolution-{context}, Server Side Conflict Detection>> for more information.
 
-Next, we must provide mutations with one extra piece of information that will be used to resolve any conflicts, by using the context. This piece of information is the `returnType` parameter. This parameter defines the Object type being operated on.
+You must provide mutations with the context, that is the `returnType` parameter to resolve conflicts.
```

Comment:
```suggestion
If you want to use conflict resolution, you must provide the client with the `returnType` of the object being operated on. This can be done in one of two ways.
```

Reply:
The wording here might not be 100% but the `context` is something they could provide in other scenarios (or for their own use cases) so we should say `returnType` is something else you can provide the `context` with.

Reply:
updated, wdyt?